installer/bundle/bundle_skel.sh

```diff
@@ -830,13 +830,18 @@ if [ "$installMode" = "I" -o "$installMode" = "U" ]; then
     fi
     curl_installed
     if [ $? -ne 0 ]; then
+        echo "curl was not found, attempting to install curl..."
+        install_extra_package curl
+    fi
+    check_program_in_path curl
```

Comment:
That is the point. You were just proposing to check them both in curl case  like it was done for sed.
I did not make sed and tar changes, and they are not related to this PR. I am going to leave it to you to make the changes and test that whenever you have time since you owned it. thanks.